common/components/SendButtonFactory/OfflineBroadcast.tsx

```diff
@@ -20,7 +20,7 @@ export const OfflineBroadcast = connect((state: AppState) => ({ offline: getOffl
 const BroadCast: React.SFC<{}> = () => (
   <p>
     To broadcast this transaction, paste the above into the{' '}
-    <Link to="pushTx">Broadcast Transaction tab</Link> or{' '}
+    <Link to="pushTx">Schedule Transaction tab</Link> or{' '}
```

Comment:
Hey, are you sure about this change? Remember you're changing original component not a cloned/copied and renamed one.

Reply:
Yeah this is an invalid change. Changed that when I was making the screenshots. Will undo.